hnn_core/gui/gui.py

```diff
@@ -1026,6 +1082,10 @@ def add_drive_widget(drive_type, drive_boxes, drive_widgets, drives_out,
                      sync_evinput=False):
     """Add a widget for a new drive."""
 
+    # Check only adds 1 tonic input widget
+    if drive_type == "Tonic" and not _is_valid_add_tonic_input(drive_widgets):
+        return
```

Comment:
Does this just mean the add drive button doesn't do anything once you add a tonic drive?

Reply:
That is correct.
@gtdang